proposals/vertical-rhythm.md

```diff
@@ -0,0 +1,97 @@
+# Vertical rhythm
+
+## Current implementations
+
+We have two main ways to control vertical rhythm: our `spacing` utility and the `<Section />` component.
+
+The `spacing` utility takes one of five string values (`xl`, `lg`, `md`, `sm`, `none`) or a default and returns css with `margin-bottom`.
+
+For example, calling `spacing('md')` returns the following CSS: `margin-bottom: 20px;`
+
+The `<Section />` component is simply a `div` with two variants: `section` and `semi-section`.
+
+The former gives the `div` a `margin-bottom` of `100px` and the latter of `50px` (these are for desktop, and values vary on other devices).
+
+## Problems
+
+### `margin-top` on the first section
+
+If a layout consists of sections with 100px of space above and below, using tools that only tackle `margin-bottom` we are unable to provide the spacing at the top of the layout.
+
+### `spacing` is a confusing name
+
+Our utility currently only adds `margin-bottom`. Spacing could imply `margin` **_or_** `padding` in **_any_** direction.
+
+### absolute values for `margin`
+
+While we generally limit the use of absolute values for `margin-bottom` in `festicket-app` by using the `spacing` utility, the many exceptions and **all** values for `margin-top` remain absolute and therefore difficult to maintain.
+
+## Proposal
+
+We maintain the `spacing` utility and `<Section />` component, but with modifications.
+
+The `spacing` module will be renamed and will allow an optional second argument to specify the direction of the margin.
+
+```js
+function rhythm(size, direction = 'bottom') {
+  try {
+    if (!['top', 'bottom'].includes(direction)) {
+      throw `direction must be either 'top' or 'bottom'`;
```

Comment:
Probably want to throw an actual error here, as it has a stack.

Reply:
I've redone the error handling as a whole. Please can you check I've got the right idea?